src/converse-chatboxviews.js

```diff
@@ -159,7 +159,10 @@ converse.plugins.add('converse-chatboxviews', {
              * @event _converse#chatBoxViewsInitialized
              * @example _converse.api.listen.on('chatBoxViewsInitialized', () => { ... });
              */
+            _converse.api.trigger('chatBoxViewPortCalculate');
+            _converse.api.trigger('chatBoxViewPortCaclulateWhenResize');
```

Comment:
This is not correct. When the user resizes the viewport it won't trigger this custom event that you've made. You need to listen for the `resize` event, like it's shown in the article I linked to earlier.